Dockerfiles/agent/entrypoint/50-ecs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Set a default config for ECS if found
+# Don't override /etc/datadog-agent/datadog.yaml if it exists
+
+if [[ -z "${ECS}" ]]; then
```

Comment:
I just merged #1182 can you reflect https://github.com/DataDog/datadog-agent/pull/1182/files#diff-4e85888650e20368a39a714841775b5d changes here? (rename to ECS_FARGATE + delete configs)